action.yml

```diff
@@ -119,6 +119,11 @@ runs:
         echo "::set-output name=docker::${docker[@]}"
         echo "::set-output name=path::$data_path"
 
+    - shell: bash
+      run: |
+        apt-get install -y qemu binfmt-support qemu-user-static
+        docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
```suggestion
    - name: Set up QEMU
      uses: docker/setup-qemu-action@v1.0.1
```

Reply:
Can't use `uses` in a composite action yet

Reply:
Oh right 🤦